client/src/containers/ChooseCampaign.js

```diff
@@ -37,15 +37,22 @@ class ChooseCampaign extends Component {
     return (
       nearbyCampaignsArr.map(c => (
         <li className="chooseCampaign-item" key={c.street_address}>
-              
+
           <input
-              type="radio"
-              id={c.street_address}
-              value={c.street_address}
-              checked={selectedOption && selectedOption.street_address === c.street_address}
-              onChange={handleOptionChange}
-            />
-            <label htmlFor={c.street_address}>
+            type="radio"
+            id={c.street_address}
+            value={c.street_address}
+            checked={selectedOption && selectedOption.street_address === c.street_address}
+            onChange={handleOptionChange}
+          />
+          <label htmlFor={c.street_address}>
+            { c.name
+              ? <div>
+                {c.name}
+                <br />
+              </div>
+              : ""
+            }
```

Comment:
This looks great!  I think we might be able to do some conditional rendering React magic here if you feel inspired by this: https://reactjs.org/docs/conditional-rendering.html ✨